### PR TITLE
Make nightly update message quieter

### LINKED ISSUE #991: `nightly` channel can be updated message is too verbose
When launching `julia +nightly`, this message is almost always displayed:

```
A new version of Julia for the `nightly` channel is available. Run:

  juliaup update

to install the latest Julia for the `nightly` channel.
```

I think this message should be shorter because
- Folks using `nightly` are typically more experienced and need less detailed prompting
- nightly is updateable several times per day so this message will show more often than not unless folks update a couple of times a day.
- there is no way to opt-out
- It's typically fine to be several versions behind on the nightly channel [added in edit]

I propose

```
A new `nightly` version is available. Install with `juliaup update`.
```

### PR DESCRIPTION
**Pull request by LilithHafner**
Fixes #991

### CODE REVIEW

**Comment by christiangnrd** [comment]
From #991:
> I think this message should be shorter because
> 
> * Folks using `nightly` are typically more experienced and need less detailed prompting
> * nightly is updateable several times per day so this message will show more often than not unless folks update a couple of times a day.
> * there is no way to opt-out

I feel like those points also apply to PR channels, should this new shorter message also show for those?

File: src/bin/julialauncher.rs [hunk 1]
```diff
@@ -235,14 +235,24 @@ fn get_julia_path_from_channel(
...
             if local_etag != server_etag {
-                eprintln!(
-                    "A new version of Julia for the `{}` channel is available. Run:",
-                    channel
-                );
-                eprintln!();
-                eprintln!("  juliaup update");
-                eprintln!();
-                eprintln!("to install the latest Julia for the `{}` channel.", channel);
+                if channel == "nightly" {
```
Comment (davidanthoff) [inline src/bin/julialauncher.rs:238]: Should also check for the arch versions, i.e. `nightly~x64` etc.

**Comment by LilithHafner** [comment]
You're quite right that those three points apply to PRs.

However, I see a fourth point that does not apply to PR branches:

- It's typically fine to be several versions behind on the nightly channel

If I am reviewing a PR and testing an out of date version of that PR, I want a large warning/hint.

**Comment by christiangnrd** [comment]
@LilithHafner I hadn't considered that at the time but I agree. PR channel requests should keep the large hint.

**Review by davidanthoff — APPROVED** [review]

**Pull request merged**